numba/cuda/cudadrv/devicearray.py

```diff
@@ -285,12 +300,28 @@ def is_f_contiguous(self):
         '''
         return self._dummy.is_f_contig
 
+    @property
+    def flags(self):
```

Comment:
You may need to add this for the `DeviceRecord` class.  There are tests failing due to missing `flags` for it.

Reply:
Which tests? The github CI builds passed?

Reply:
The TravisCI builds cannot test GPU code.  So, the GPU tests are skipped on CI.  To run all the GPU tests, use `python runtests.py -v numba.cuda.tests`.  

The failing tests were from `numba.cuda.tests.cudapy.test_record_dtype`.  They are passing now.